ignite/templates/module/create/files/base/x/{{moduleName}}/simulation/placeholder.go.plush

```diff
@@ -0,0 +1,3 @@
+package simulation
+
+const PlaceHolder = "placeholder"
```

Comment:
Are we using this only to avoid the import issue into the `module/simulation.go`? Because we can use the `xast` package to add a new import every time, the method checks if the import already exists to avoid duplicated imports. We can also create an issue by removing all blank variables from the project. We don't need to worry about it anymore

Reply:
True, I forgot about that package, you are right, that was the reason.

Reply:
are you going to remove it in https://github.com/ignite/cli/issues/4548 follow-up @Pantani?

Reply:
yes, I will do for the other variables, I can do for this one